packages/manager/apps/telecom/src/app/telecom/pack/resiliation/pack-resiliation.js

```diff
@@ -1,13 +1,9 @@
 angular.module('managerApp').config(($stateProvider) => {
   $stateProvider.state('telecom.packs.pack.resiliation', {
     url: '/resiliation',
-    views: {
-      'packView@telecom.packs': {
-        templateUrl: 'app/telecom/pack/resiliation/pack-resiliation.html',
-        controller: 'PackResiliationCtrl',
-        controllerAs: 'PackResiliation',
-      },
-    },
+    templateUrl: 'app/telecom/pack/resiliation/pack-resiliation.html',
```

Comment:
use template instead of templateUrl

Reply:
@radireddy I understand your point but this requires a huge rework on top of what has already been done so I think it is out of scope here